cishell/src/main/java/org/cishell/cibridge/cishell/impl/CIShellCIBridgeDataFacade.java

```diff
@@ -44,6 +48,20 @@ public CIShellCIBridgeDataFacade() {
         });
         dataUpdatedObservable = dataupdatedobservable.share().publish();
         dataUpdatedObservable.connect();
+
+        Observable<org.cishell.cibridge.core.model.Data> dataaddedobservable = Observable.create(emitter -> {
+            dataAddedObservableEmitter = emitter;
+
```

Comment:
Please take a pass to remove extraneous blank lines. The test for this is: does this blank line make it easier to understand the code? If yes, keep it, otherwise remove it. For instance
```java
public foo() {

    doSomething();

}
```

has no business having all those extra lines. it does not aid in understanding. this should be:

```java
public foo() {
    doSomethng();
}
```